bower.json

```diff
@@ -5,41 +5,41 @@
   "authors": [
     "The Polymer Authors"
   ],
-  "private": true,
-  "homepage": "https://github.com/PolymerElements/app-layout",
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "homepage": "https://elements.polymer-project.org/elements/app-layout",
```

Comment:
Why are we changing the homepage url in app-layout?  All the other iron/paper/app elements are pointing to github instead of polymer catalog.  We should be consistent.
cc @blasten